packages/gafl-webapp-service/src/handlers/page-handler.js

```diff
@@ -43,9 +43,8 @@ const getBackReference = async (request, view) => {
   if (!current || !current.backLink) {
     return null
   }
-
   if (typeof current.backLink === 'function') {
-    const backLink = current.backLink(
+    const backLink = await current.backLink(
```

Comment:
When you come across a bug, best practice is that you should write a test that demonstrates the bug (i.e. fails), then write code to make that test pass. That way, the bug will never occur again, as there's a test that catches it.